.github/workflows/build-and-deploy.yml

```diff
@@ -16,31 +17,27 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      # - name: Docker meta
-      #   id: docker_meta
-      #   uses: crazy-max/ghaction-docker-meta@v1
-      #   with:
-      #     images: ghcr.io/viet-aus-it/discord-bot
-      #     tag-sha: true
-      #     tag-custom: production
-
-      # - name: Setup Docker Buildx
-      #   uses: docker/setup-buildx-action@v1
-
-      # - name: Login to GitHub Docker
-      #   uses: docker/login-action@v1
-      #   with:
-      #     registry: ghcr.io
-      #     username: ${{ secrets.USER_NAME }}
-      #     password: ${{ secrets.ACCESS_TOKEN }}
-
-      # - name: Build and Push docker image
-      #   uses: docker/build-push-action@v2
-      #   with:
-      #     context: discord-bot
-      #     file: discord-bot/.docker/Dockerfile
-      #     pull: true
-      #     push: true
-      #     tags: ${{ steps.docker_meta.outputs.tags }}
-      #     target: production
-
+      - name: Configure SSH client
+        run: |
+          mkdir -p ~/.ssh/
+          echo "$SSH_KEY" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          cat >> ~/.ssh/config <<END
+          Host *
+          ForwardAgent yes
+          StrictHostKeyChecking no
+          UserKnownHostsFile /dev/null
+          END
+        env:
+          SSH_KEY: ${{ secrets.SSH_KEY_EC2_CHATBOT }}
+
+      - name: SSH to VAIT server
+        run: |
+          cmd="/usr/local/bin/deploy-chatbot ${SSH_USER} ${{ github.event.repository.name }} ${GITHUB_REF##*/} /${{ github.event.repository.name }}/discord-bot/.env.docker.${GITHUB_REF##*/},discord-bot,.env.docker /${{ github.event.repository.name }}/discord-bot/.env.${GITHUB_REF##*/},discord-bot,.env"
```

Comment:
cái script này hiện tại có gì? Nếu nó có docker build, thì em nghĩ chạy cái build & publish trên này ổn hơn, rồi sau đó khi deploy thì con EC2 pull image về thôi, đỡ tốn network bandwidth khi build + mình actually track được build có lỗi hay không ở đây.

Reply:
bữa anh nhớ em nói build&publish ở trên này rồi(??)
deploy-chatbot
* clone branch
* pull env files
* copy files to destination
* restart containers

Reply:
anh sẽ có 1 task khác để viết Terraform. Deploy script sẽ để trên S3 hoặc TF

Reply:
chưa có pipeline build & publish docker image anh ơi :))